common/djangoapps/course_modes/tests/test_signals.py

```diff
@@ -87,3 +89,28 @@ def test_verified_mode_explicitly_set(self, verification_window):
             course_mode.refresh_from_db()
 
             self.assertEqual(course_mode.expiration_datetime, self.end - timedelta(days=verification_window))
+
+    def test_masters_mode(self):
+        # create an xblock with verified group access
+        verified_section = ItemFactory.create(
+            category="sequential",
+            metadata={'group_access': {50: [2]}}
```

Comment:
Is there a way to make this this not rely on these values and refer to `ENROLLMENT_TRACK_PARTITION_ID` and `getattr(settings, 'COURSE_ENROLLMENT_MODES', {}).get('verified', {}).get('id', None)`? That might make it clearer what those numbers represent, too. I find that I have to take extra steps to work my way backwards from, "Okay, what is 50? Oh, that's the partition ID, so the array represents the ids of the course modes. Okay, I'll go look at COURSE_ENROLLMENT_MODES to see what 1, 2, and 7 are." It would be great if the code could save me that step.

Reply:
I updated the tests to use the correct constants.